Tasks/AzureRmWebAppDeployment/operations/AzureAppServiceUtility.ts

```diff
@@ -57,9 +57,9 @@ export class AzureAppServiceUtility {
         return defer.promise;
     }
 
-    public async getApplicationURL(): Promise<string> {
+    public async getApplicationURL(virtualApplication?: string): Promise<string> {
         let webDeployProfile: any =  await this.getWebDeployPublishingProfile();
-        return await webDeployProfile.destinationAppUrl;
+        return await webDeployProfile.destinationAppUrl + "/" + (virtualApplication ? virtualApplication : "");
```

Comment:
For a virtual application, URL is not terminating with '/' but for a web app, URL is terminating with '/'. Should behaviour be same behavior in both cases?

Reply:
thanks Ajay. modified the logic.